examples/meta/generator/targets/scala.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
I think we don't need another json here, just use the existing java json, the syntax is example the same